docs/modules/publisher-commandline.md

```diff
@@ -3,8 +3,11 @@
 ### OPC Publisher Command Line Arguments for Version 2.6 and above
 
              --pf, --publishfile=VALUE
-                                       the filename to configure the nodes  to publish.
-                                       If this Option is specified it puts OPC Publisher into standalone  mode.
+                                       the filename to configure the nodes to publish.
+                                       If this option is specified it puts OPC Publisher into standalone mode.
+             --pfs, --publishfileschema=VALUE
```

Comment:
Let's create a new heading in this file for options introduced after `2.8.0` version as listing this argument under `for Version 2.6 and above` is misleading.

Reply:
Version changes will be pushed in next commit.

Reply:
@suneetnangia, I would still suggest to add a new markdown heading for newly introduced command line arguments like the one bellow. The current changes introduce a gap in between `2.5` and `2.8` which will be confusing for people reading the file and for people still using `2.6` or `2.7` versions.

`OPC Publisher Command Line Arguments Introduced After Version 2.8.0`

Reply:
in the next commit, I have put this under the 2.8.0 section. Given that 2.8.0 is an LTS it feels more natural for the additions to come after the base (e.g. LTS).  Let me know if you want it bumped to the top of the file.

Reply:
@WilliamBerryiii, can you please restore the original naming of the first paragraph that used to say `OPC Publisher Command Line Arguments for Version 2.6 and above` instead of `OPC Publisher Command Line Arguments for Version 2.8.0 and above` ? This has been changed in one of the previous commits. Everything else looks great.

Reply:
Reverted per commit: https://github.com/Azure/Industrial-IoT/pull/1305/commits/fa262e34b5e40d67d3fe8eb7c1feece3a93e6fb2

Thanks for clarification on that; and really appreciate all the support from you to land this!